weechat-android/src/main/java/com/ubergeek42/WeechatAndroid/service/SSLHandler.java

```diff
@@ -37,20 +37,8 @@ void loadKeystore() {
         try {
             sslKeystore = KeyStore.getInstance("BKS");
             sslKeystore.load(new FileInputStream(keystoreFile), KEYSTORE_PASSWORD.toCharArray());
-        } catch (KeyStoreException e) {
-            e.printStackTrace();
-        } catch (NoSuchAlgorithmException e) {
-            // Should never happen
-            e.printStackTrace();
-        } catch (CertificateException e) {
-            // Ideally never happens
-            e.printStackTrace();
-        } catch (FileNotFoundException e) {
-            logger.debug("Keystore not found, creating...");
-            createKeystore = true;
```

Comment:
Looks like we probably still need this Catch for FileNotFoundException(so that createKeystore can be set to true)